include/deal.II/dofs/dof_accessor.h

```diff
@@ -2019,6 +2019,15 @@ class DoFCellAccessor : public DoFAccessor<DoFHandlerType::dimension,
   void
   clear_p_coarsen_flag() const;
 
+  /**
+   * Returns the active_fe_index that this cell would have if refinement would
+   * have happened. Prediction is based on the current p-refinement flags set on
```

Comment:
I would rather point to the function that is attached to FECollection that determines what's the next/previous FE index (which is exactly what you do inside).